angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -99,7 +105,13 @@ import {
             >
               <div
                 *ngFor="let item of localLinks"
-                class="sprk-o-Stack__item sprk-o-Stack__item--third@m sprk-o-Box sprk-u-PaddingRight--a sprk-o-Stack sprk-o-Stack--large"
+                class="
+                  sprk-o-Stack__item
+                  sprk-o-Stack__item--third@m
+                  sprk-c-Footer__local-links
```

Comment:
Could not use the <sprk-stack> component here because of angular custom elements but we will look at it more closely in the API upgrade work for footer. Should be something we can do.